### PR TITLE
[tune] Fix bug in example where config hyperparameters were ignored

### PR DESCRIPTION
**Pull request by arcelien**
## What do these changes do?
A fix to an example for tune (`python/ray/tune/examples/pbt_tune_cifar10_with_keras.py`) where the hyperparameters for the optimizer, learning rate and decay, were not being passed into the optimizer. 

This means that the current optimizer uses default values for the hyperparameters no matter the config.

### CODE REVIEW

**Review by richardliaw — APPROVED** [review]
Nice!
